act-rules-format/act-rules-format.bs

```diff
@@ -563,10 +577,10 @@ Test cases are (snippets of) content that can be used to validate the implementa
 </aside>
 
 
-Change Log {#change-log}
+Rule Versions {#rule-versions}
 ------------------------
 
-It is important to keep track of changes to the ACT Rules so that users of the rules can understand if changes in test results are due to changes in the rules used when performing the tests, or from changes in the content itself. All changes to an ACT Rule that can change the [=outcomes=] of an evaluation <em class="rfc2119">must</em> be recorded in a change log. The change log can either be part of the rule document itself or be referenced from it.
+It is important to keep track of versions of ACT Rules so that users of the rules can understand if changes in test results are due to changes in the rules used when performing the tests, or from changes in the content itself. All previous versions of an ACT Rule  <em class="rfc2119">must</em> be recorded either in the rule document or referenced from it.
```

Comment:
```suggestion
It is important to keep track of versions of ACT Rules so that users of the rules can understand if changes in test results are due to changes in the rules used when performing the tests, or from changes in the content itself. All previous versions of an ACT Rule <em class="rfc2119">must</em> be recorded either in the rule document or referenced from it.
```